samwise.js

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.